### ISSUE TITLE
Error messages when opening project (after using aaa)

### ISSUE THREAD

**Issue by yoda321** [radare/radare2#8582]
When I open a project r2 -p mypoject I get a bunch of errors with some numbers:

> 0
> 0
> Cannot add reference to non-function
> Cannot add reference to non-function
> 1482

I use latest version of radare2 from git repository on Arch Linux. This is list of steps to reproduce this bug.

> r2 /bin/ls
> aaa
> Ps myproject
> q
> r2 -p myproject

**Comment by Maijin** [comment]
Hello,

Ensure you are using radare2 from git, if you're unsure paste output of `r2 -v` here.
To install radare2 from git, first uninstall your version of radare2 and clean your distro. Then use `git clone https://github.com/radare/radare2 && cd radare2 && ./sys/install.sh`, verify your version and check if there is no error using `r2 -v`.

**Comment by yoda321** [comment]
Output of r2 -v
>radare2 1.7.0-git 16152 @ linux-x86-64 git.1.6.0-928-gfa21d5702
>commit: fa21d5702935cb733e900154d0a15e48ad8391d9 build: 2017-09-25__11:34:41

I also modified line 2388 of libr/core/cmd_anal.c for debugging.
> eprintf ("Cannot add reference to non-function: %s\n", input);
And I tried running some of commands (on file and on project). For example:
>afxC 0x1471b 0x21fd50
It executes normally.

**Comment by saruman9** [comment]
It opens project with the same errors, even when not used `aaa`.

**Comment by Maijin** [comment]
```
0
0
```

Is fixed the:

```
Cannot add reference to non-function
```

Still found time to time

**Comment by radare** [comment]
and now?

**Comment by Maijin** [comment]
"The Cannot add reference to non-function" is still there

**Comment by Maijin** [comment]
r2 /bin/ls
aaa
Ps proj
q
r2 -p proj

**Comment by Maijin** [comment]
Basically it says "No file selected" when typing `i`

**Comment by radare** [comment]
this?

<img width="612" alt="screen shot 2018-01-28 at 00 02 48" src="https://user-images.githubusercontent.com/917142/35477198-9adf92e2-03be-11e8-8459-bce0431df064.png">

**Comment by Maijin** [comment]
Try i

**Comment by radare** [comment]
yeah because projects are broken and the bin info is not reloaded from the bin, but from the flags and sections info. We should add r_bin_obj_dump/restore to keep the state of all the rbinobjects loaded in a session and then loading them from the project. not a small task and needs rbin to be refactored (condret is working on that)
